### ISSUE TITLE
command call and caps

### ISSUE THREAD

**Issue by jonaprieto** [FlamingTempura/bibtex-tidy#22]
Hi,

It will be nice if on the website one can get the command call to produce
the last outcome. I'm thinking to give the user a line code to run on their terminal,
so next time they can, like me, incorporate bibtex-tidy in their workflow.

Another feature request I have in mind is on an entry's key. A few options I find
convenient are: drop all caps, and generate the key. Although, the latter is way more
complex. 

Thanks, 
Great job!

**Comment by FlamingTempura** [comment]
> It will be nice if on the website one can get the command call to produce
> the last outcome. I'm thinking to give the user a line code to run on their terminal,
> so next time they can, like me, incorporate bibtex-tidy in their workflow.

Great idea!

> Another feature request I have in mind is on an entry's key. A few options I find
> convenient are: drop all caps, and generate the key. Although, the latter is way more
> complex.

Did you try the drop all caps option?
![image](https://user-images.githubusercontent.com/1085434/106042884-8c1ddd00-60d5-11eb-9856-5c033b35d820.png)

Interesting idea generating the key, I guess it would be of the format `<author><date>`?

**Comment by jonaprieto** [comment]
Yeah, that key generation criteria can be the default. Maybe later one can let the user uses their own format.

Ops, I didn't try that option of drop all caps. I will check that out.

**Comment by norro** [comment]
google scholar generates the key in the format `<author><year><first word of title>`, which leads to less redundant keys, I guess.

**Comment by FlamingTempura** [comment]
> It will be nice if on the website one can get the command call to produce
the last outcome. I'm thinking to give the user a line code to run on their terminal,
so next time they can, like me, incorporate bibtex-tidy in their workflow.

You should now see the CLI command in the sidebar:
![image](https://user-images.githubusercontent.com/1085434/111882279-b3de4280-89ac-11eb-95c6-eeda553e2406.png)

**Comment by jonaprieto** [comment]
![image](https://user-images.githubusercontent.com/1428088/114788601-0a5b5880-9d82-11eb-9154-c61ab1990df1.png)

I tried other examples and after copying and paste, in the shell the commands don't work.
Also, it generates commands options that seem to not be available, e.g. --no-enclosing-braces

**Comment by FlamingTempura** [comment]
I've just pushed an update that fixes the incorrect arguments that were being generated. Could you give it a try?
